lineman/app/services/mention_service.coffee

```diff
@@ -0,0 +1,7 @@
+angular.module('loomioApp').factory 'MentionService', ->
```

Comment:
Yep, I think I know what's going on. I was talking about the serializer modifying the body before the client gets it.

If we serialized the mentioned usernames into a comment property then this might be able to be done client side without modifying the text for edit.. IE: you could just have renderMentions and edit would not need anything.

Aside:
Will this work ok with the change detection, "are you sure you want to cancel?" stuff?